### PR TITLE
feat(clientLibs): add Ruby client library to Client Page

### LINKED ISSUE #16517: Add Ruby Client Docs to Client Page
We have a new ruby client for influxdb 2.0. We should add similar docs to the app.
https://github.com/bonitoo-io/influxdb-client-ruby

@alexpaxton need a ruby logo

### PR DESCRIPTION
**Pull request by bednar**
Closes #16517 

* add Ruby client library to Client Page
* add actual version of Java client

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by drdelambre** [comment]
NOTE: still waiting on the ruby logo (it's currently an empty SVG)

**Comment by mavarius** [comment]
Logo added in https://github.com/influxdata/influxdb/pull/16655

File: ui/src/clientLibraries/components/ClientRubyOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+const mstp = (state: AppState): StateProps => {
+  const {id} = getOrg(state)
+
+  return {
+    org: id,
+  }
+}
+
+export {ClientRubyOverlay}
+export default connect<StateProps, {}, Props>(
```
Comment (drdelambre) [inline ui/src/clientLibraries/components/ClientRubyOverlay.tsx]: i think the last parameter on this is supposed to be `{}` as it represents the all the props being passed in that aren't part of state and dispatch props, but if the typescript checks are happy, i'm happy

**Comment by russorat** [comment]
LGTM
image

**Pull request merged**